scripts/azure-ci.yml

```diff
@@ -0,0 +1,5 @@
+trigger:
```

Comment:
These yml files are what we can point AzDO towards.

Then, ideally we would enable them only for `feature/ci` (or some branch) while I get dependencies for the hosted pools worked out. Then, one that PR is merged, we can enable CI globally.